reconcile/utils/aws_api.py

```diff
@@ -1031,9 +1031,13 @@ def create_cloudwatch_tag(self, account, group_name, new_tag):
         cloudwatch_logs.tag_log_group(logGroupName=group_name, tags=new_tag)
 
     def get_cloudwatch_logs(self, account):
+        log_group_list = []
         cloudwatch_logs = self._account_cloudwatch_client(account["name"])
-        log_groups = cloudwatch_logs.describe_log_groups()["logGroups"]
-        return log_groups
+        paginator = cloudwatch_logs.get_paginator("describe_log_groups")
+        for page in paginator.paginate():
+            for log_group in page["logGroups"]:
+                log_group_list.append(log_group)
```

Comment:
tiny nit: you could go for `log_group_list.extend(page["logGroups"])` to avoid the additional loop